### ISSUE TITLE
Dotnet 7 Version

### ISSUE THREAD

**Issue by renerlemes** [jincod/dotnetcore-buildpack#184]
Is possible to use with dotnet 7? I see in the code, that version 6 is fixed. Is possible to change to detect the version and install de runtime and/or sdk to run application?

**Comment by jincod** [comment]
Yes, you can. Please see https://github.com/jincod/dotnetcore-buildpack/issues/131#issuecomment-726409101

I'll update to 7 soon

**Comment by conraduouou** [comment]
This may be a newbie question, but how do I exactly set the environment variables to version 7 and use it as a buildpack? What's being used in the latest is still version 6.0....

**Comment by jincod** [comment]
Hi @conraduouou 

Points to the latest .net 7 version
```
heroku buildpacks:set https://github.com/jincod/dotnetcore-buildpack
```

You can set the exactly version you want:
```
heroku config:set DOTNET_SDK_VERSION=7.x.x.
heroku config:set DOTNET_RUNTIME_VERSION=7.x.x
```

All versions are here https://dotnet.microsoft.com/en-us/download/dotnet/7.0
